app/constants/line-descriptions.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  G00: 'G00 - Gross value of claim',
+  P01: 'P01 - Declaration penalty applied to Young Farmer',
```

Comment:
I just wanted to clarify, but it's definitely 'Young Farmer' and not 'Young Farmers'?

Reply:
Will update. I used P01 in a test and thought it was tidy to keep them all together